### ISSUE TITLE
Evaluation metrics 

### ISSUE THREAD

**Issue by di-mi-ta** [yxuhan/IALS#1]
Hi, 

This work is interesting!

Could you release the evaluation metrics code to guarantee fair comparison? 

Thanks!

**Comment by yxuhan** [comment]
Hi @di-mi-ta 

Do you mean the code for DT metric?

**Comment by di-mi-ta** [comment]
Yes! 

Btw, I wonder whether you trained the new classifiers for attribute classification or you converted weights from these pretrained models (https://drive.google.com/drive/folders/1MvYdWCBuMfnoYGptRH-AgKLbPTsIQLhl) from tensorflow to pytorch! 
If you convert weights, could you release the code? I would like to try with some other attributes different from the attributes you released! 

Thanks!

**Comment by yxuhan** [comment]
@di-mi-ta 

We trained a new classifier for attribute classification to evaluate the DT metric, but we think using these pre-trained models (https://drive.google.com/drive/folders/1MvYdWCBuMfnoYGptRH-AgKLbPTsIQLhl) is also fine. 

Set lambda_1 near 1 (e.g. 0.75) and lambda_2 near 0 is an empirically good initial choice for editing other attributes. 

We will release the code for the DT metric in the future and notify you as soon as we make it.

**Comment by di-mi-ta** [comment]
Thanks :), I hope you releasing it soon!
